go.mod

```diff
@@ -3,18 +3,23 @@ module github.com/aerospike/backup
 go 1.21
 
 require (
+	github.com/aerospike/aerospike-client-go/v7 v7.1.0
 	github.com/aws/aws-sdk-go-v2 v1.24.0
 	github.com/aws/aws-sdk-go-v2/config v1.26.1
 	github.com/aws/aws-sdk-go-v2/service/s3 v1.47.5
 	github.com/aws/smithy-go v1.19.0
 	github.com/prometheus/client_golang v1.18.0
 	github.com/reugn/go-quartz v0.10.0
 	github.com/spf13/cobra v1.8.0
+	github.com/swaggo/swag v1.16.2
```

Comment:
remove this dependency

Reply:
it was added by go mod tidy, maybe leave it?